### ISSUE TITLE
Option to only show key value (not name) in breadcrumb for collections

### ISSUE THREAD

**Issue by gordon-matt** [GregFinzer/Compare-Net-Objects#274]
Is there an option to not show the key name (only the value) in breadcrumbs for collections?

This is an example of what I am seeing:

```
Tables[TableName:Commission].Fields[DestinationField:AdviserCode].Enabled
```

But I would prefer to see it as follows:

```
Tables[Commission].Fields[AdviserCode].Enabled
```

An option could be as follows:

```
logic.Config.ShowCollectionKeyName = false;
```

For multi-key collections, I guess you could separate the values with a comma or some other separator.

Maybe even have a separator option, such as:

```
logic.Config.CollectionKeySeparator = '|'; // Default = ','
```

Example results: `Tables[MyTable,true]` or `Tables[MyTable|true]`

**Comment by GregFinzer** [comment]
That feature does not exist.  If you would like to implement I can approve the pull request.

**Comment by GregFinzer** [comment]
Merged

**Comment by gordon-matt** [comment]
Sweet. When should I expect the new package?

**Comment by GregFinzer** [comment]
There is another PR that is not compiling.  I am waiting on another person to fix that.
